include/mbgl/storage/default_file_source.hpp

```diff
@@ -174,6 +174,11 @@ class DefaultFileSource : public FileSource {
      */
     void put(const Resource&, const Response&);
 
+    /*
+     * Delete existing database and re-initialize.
+     */
+    void clearCache();
```

Comment:
maybe `recreateCache()` or `resetCache()`?

Reply:
renamed to  `resetCache()`